packages/shared/src/components/auth/OnboardingRegistrationForm.tsx

```diff
@@ -247,4 +291,113 @@ const OnboardingRegistrationForm = ({
   );
 };
 
+export const TermAndPrivacy = (): ReactElement => (
+  <Typography color={TypographyColor.Secondary} type={TypographyType.Footnote}>
+    By continuing, you agree to the{' '}
+    <Typography
+      className="underline hover:no-underline"
+      color={TypographyColor.Tertiary}
+      href={termsOfService}
+      rel="noopener"
+      tag={TypographyTag.Link}
+      target="_blank"
+    >
+      Terms of Service
+    </Typography>{' '}
+    and{' '}
+    <Typography
+      className="underline hover:no-underline"
+      color={TypographyColor.Tertiary}
+      href={cookiePolicy}
+      rel="noopener"
+      tag={TypographyTag.Link}
+      target="_blank"
+    >
+      Privacy Policy
+    </Typography>
+    .
+  </Typography>
+);
+
+const AlreadySignedLogin = ({ onClickLogin }: { onClickLogin: () => void }) => (
```

Comment:
Let's move this to its own file to make it reusable, if I'm not mistaken, this exists already.

Reply:
Moved to the already existing component, thanks! 🙏